src/main/jhipster/src/main/webapp/app/crop-settings-manager/parameters/parameters-pane.component.html

```diff
@@ -11,15 +11,21 @@
 			<tbody>
 			<tr *ngFor="let cropParameter of cropParameters; index as index">
 				<td>{{cropParameter.key}}</td>
-				<td class="editable">
+				<td class="editable input-group">
```

Comment:
In the UI, If the crop parameter value is blank, it's hard to know where to click to edit it. Something to improve.